### DISCUSSION TITLE
Question: asciidoc and/or reStructuredText support (👍49)

### DISCUSSION THREAD

**Discussion by IlCallo** [11ty/eleventy#3981] [Enhancement Queue]
Are you planning to support Asciidoc or reStructuredText in the short time?
They are more expressive and extensible than MarkDown (at least in it's CommonMark which tries to standardise the language) but near as simple, as far as I know.
Just wondering if you were thinking about them alongside with all the others templating languages you suppot

**Comment by zachleat** [comment]
In the short term? I don’t have any plans to do this short term but am open to the voting will of the people 😎

**Comment by zachleat** [comment]
This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

The enhancement backlog can be found here: https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+

Don’t forget to upvote the top comment with 👍!

**Comment by zachleat** [comment]
You may also want to upvote #117 and/or #118, which would allow others to add plugins for arbitrary types

**Comment by IlCallo** [comment]
Gotcha, ty

**Comment by zachleat** [comment]
https://www.npmjs.com/package/asciidoctor.js

https://twitter.com/stoyanstefanov/status/1085331112346316800

**Comment by saneef** [comment]
If anyone still looking to use Asciidoc with Eleventy, I have created a plugin [`eleventy-plugin-asciidoc`](https://github.com/saneef/eleventy-plugin-asciidoc). It needs the new [Eleventy v1.x Beta](https://www.11ty.dev/blog/eleventy-v1-beta/)

**Comment by rmschindler** [comment]
hi @saneef, thanks for creating the eleventy-plugin-asciidoc . I’m trying to get it to work with a barebones Eleventy project which I just started — could you share any repos that have it working?

**Comment by saneef** [comment]
@rmschindler I hope https://github.com/saneef/eleventy-plugin-asciidoc-tester helps.
